src/BloomBrowserUI/pageChooser/page-chooser.ts

```diff
@@ -249,15 +249,25 @@ class PageChooser {
             
             // Grab all pages in this group
             // N.B. normal selector syntax or .find() WON'T work here because pageData is not yet part of the DOM!
-            var pages = $(pageData).filter('.bloom-page[id]').filter('[data-page="extra"]');
+            // Creating a jquery object via $(pageData) causes any img elements in the html string to be dereferenced,
+            // which can cause the Bloom server to complain about not finding files of the form "pageChooser/read.png".
+            // So we must remove the img elements from the returned string before the conversion to a jquery object.
+            // If the replace method had group selection and replacement, we could revise the src attributes into api
+            // calls, but that isn't possible in Typescript without extremely convoluted string manipulation.  See
```

Comment:
Is that right? I see http://stackoverflow.com/questions/1234712/javascript-replace-with-reference-to-matched-group, and the built-in Regex class.

But if I understand correctly, the better reason for not just "fixing" the urls is what you say below: we don't *want* those images. All that is wanted by this method, and the methods, is to find out some ids of the pages.

I recommend removing that sentence.